docs/contracts/abstract-contracts.rst

```diff
@@ -6,27 +6,31 @@
 Abstract Contracts
 ******************
 
-Contracts are marked as abstract when at least one of their functions lacks an implementation as in the following example (note that the function declaration header is terminated by ``;``)::
+Contracts need to be marked as abstract when at least one of their functions was not implemented.
```

Comment:
Please add: You can also mark contracts as abstract even though all functions are implemented.